lib/Client.js

```diff
@@ -1929,13 +1930,14 @@ class Client extends EventEmitter {
     /**
     * Get a guild's data via the REST API. REST mode is required to use this endpoint.
     * @arg {String} guildID The ID of the guild
+    * @arg {Boolean} [withCounts=false] Whether the guild object will have approximateMemberCount and approximatePresenceCount
     * @returns {Promise<Guild>}
     */
-    getRESTGuild(guildID) {
+    getRESTGuild(guildID, withCounts = false) {
```

Comment:
Uh..  this is added in from #920 by accident, isn't it?

Reply:
oh, I messed up somewhere thanks